packages/victory-brush-container/src/brush-helpers.js

```diff
@@ -82,11 +82,29 @@ const Helpers = {
     };
   },
 
-  getDefaultBrushArea(defaultBrushArea, domain, cachedDomain) {
+  getDefaultBrushArea(targetProps, cachedDomain, evt) {
+    const { defaultBrushArea, domain, fullDomain, scale, horizontal } = targetProps;
     if (defaultBrushArea === "none") {
       return this.getMinimumDomain();
     } else if (defaultBrushArea === "disable") {
       return cachedDomain;
+    } else if (defaultBrushArea === "move") {
+      // eslint-disable-next-line no-unused-vars
+      const { x1, x2, y1, y2, ...restProps } = targetProps;
+      const brushBox = this.getDomainBox(restProps, fullDomain, cachedDomain);
+      const parentSVG = restProps.parentSVG || Selection.getParentSVG(evt);
+      const pannedBox = this.panBox(
+        {
+          ...restProps,
```

Comment:
You can avoid a second call to `this.getDomainBox`, and avoind needing to destructure / remove props from `restProps` like so: 

```
const pannedBox = this.panBox(
        {
          ...targetProps,
          ...brushBox,
          brushDomain: cachedDomain,
          startX: (brushBox.x1 + brushBox.x2) / 2,
          startY: (brushBox.y1 + brushBox.y2) / 2
        },
        Selection.getSVGEventCoordinates(evt, parentSVG)
      );
```